frontend/css/wi-volunteer-management-public.css

```diff
@@ -80,4 +80,33 @@
 
 #wivm-sign-up-form .field-error {
 	border: 2px solid #990000;
+}
+
+/* Widget Styles */
+.widget-volunteer-opp-info {
```

Comment:
@jg314 

**Pull request updated**:

I've removed the max-height and scrolling styling for the widget. Thanks for the feedback.